core/domain/exp_services.py

```diff
@@ -948,9 +952,12 @@ def update_exploration(
         committer_id, updated_exploration, commit_message, change_list)
 
     discard_draft(exploration_id, committer_id)
-    # Update summary of changed exploration.
-    regenerate_exploration_summary_with_new_contributor(
-        exploration_id, committer_id)
+
+    # Update summary of changed exploration in a deferred task.
+    taskqueue_services.defer(
```

Comment:
The exp_summary generation is taking 1-2 secs to complete, I've moved it to the deferred task as this doesn't need to be computed around the time of updating the exp.